euth/comments/signals.py

```diff
@@ -1,12 +1,43 @@
 from django.contrib.contenttypes.models import ContentType
-from django.db.models.signals import post_delete
+from django.db.models.signals import post_delete, post_save
 from django.dispatch import receiver
 
+from euth.actions.models import Action
 from euth.contrib import services
 
+from euth.projects.models import Project
+
 from .models import Comment
 
 
+@receiver(post_save, sender=Comment)
+def add_action(sender, instance, created, **kwargs):
```

Comment:
Shouldn't this code be in the actions app? Otherwise we are required to change all other apps in order to have actions tracked.